pkg/cmd/propertiesGet_test.go

```diff
@@ -17,11 +17,12 @@ func TestPropertiesGetCommandInCommandCollectionHasName(t *testing.T) {
 	factory := NewMockFactory()
 	commands, _ := NewCommandCollection(factory)
 
-	propertiesGetCommand := commands.GetCommand(COMMAND_NAME_PROPERTIES_GET)
+	propertiesGetCommand, err := commands.GetCommand(COMMAND_NAME_PROPERTIES_GET)
```

Comment:
if you get an err, you probs need to check that not being nil first ?
Else that's not the error which will show up most prominently when the testcase fail is reported.